generator-torsion/generators/app/templates/_d8/_cmm_torsion.info.yml

```diff
@@ -16,6 +16,14 @@ stylesheets-remove:
   - core/assets/vendor/normalize-css/normalize.css
   - core/modules/system/css/system.theme.css
 
+component-libraries:
+  component:
+    paths:
+      - _patterns/02-component
+  layout:
+    paths:
+      - _patterns/04-layout
+
```

Comment:
the _patterns could probably be set as a variable with yeoman depending on where we're creating the component folder